code/__DEFINES/access.dm

```diff
@@ -4,7 +4,7 @@ It's best not to mess with the numbers of the regular access levels because
 most of them are tied into map-placed objects. This should be reworked in the future.*/
 //WE NEED TO REWORK THIS ONE DAY.  Access levels make me cry - Apophis
 #define ACCESS_MARINE_SENIOR 1
-#define ACCESS_MARINE_DATABASE 2
+#define ACCESS_MARINE_GENERAL 2
```

Comment:
This is likely going to impact things on maps that are manually set, and also doesn't have a purpose to be general access.